Makefile

```diff
@@ -9,7 +9,13 @@ PROJ = google_containers
 
 PKG_SOURCES := $(shell find pkg -name '*.go')
 
-node-problem-detector: $(PKG_SOURCES) node_problem_detector.go
+vet:
+	go list ./... | grep -v "./vendor*" | xargs go vet
```

Comment:
What `.` means is that it means anything prefixing vendor directory. Because we run `go list ./...` in the root directory, it output
```
k8s.io/node-problem-detector
k8s.io/node-problem-detector/pkg/condition
k8s.io/node-problem-detector/pkg/kernelmonitor
k8s.io/node-problem-detector/pkg/kernelmonitor/translator
k8s.io/node-problem-detector/pkg/kernelmonitor/types
k8s.io/node-problem-detector/pkg/kernelmonitor/util
k8s.io/node-problem-detector/pkg/problemclient
k8s.io/node-problem-detector/pkg/problemdetector
k8s.io/node-problem-detector/pkg/types
k8s.io/node-problem-detector/pkg/util
k8s.io/node-problem-detector/pkg/util/nethealth
k8s.io/node-problem-detector/vendor/code.cloudfoundry.org/clock
k8s.io/node-problem-detector/vendor/code.cloudfoundry.org/clock/fakeclock
k8s.io/node-problem-detector/vendor/github.com/blang/semver
k8s.io/node-problem-detector/vendor/github.com/coreos/go-oidc/http
```

What i mean is to match`.*/vendor/.*` in Golang regexp expression.